setup.py

```diff
@@ -8,6 +8,9 @@
 import os
 import platform
 import shutil
+import pandas as pd
```

Comment:
lgtm but not sure about a vs b?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
for test

Reply:
Logic should in general be asking the question "Is this true?" instead of "Is this not true?"

📄 [Principles.dev](https://principles.dev/p/)

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Secure, Inelegant